Chef.PowerShell.sln

```diff
@@ -29,10 +29,10 @@ Global
 		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Debug|x64.Build.0 = Debug|Any CPU
 		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Debug|x86.Build.0 = Debug|Any CPU
-		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Release|Any CPU.Build.0 = Release|Any CPU
-		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Release|x64.ActiveCfg = Release|Any CPU
-		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Release|x64.Build.0 = Release|Any CPU
+		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Release|Any CPU.ActiveCfg = Release|x64
+		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Release|Any CPU.Build.0 = Release|x64
+		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Release|x64.ActiveCfg = Release|x64
+		{AB46BA32-444C-4776-BEE4-C41BFDF354C3}.Release|x64.Build.0 = Release|x64

```

Comment:
Why's this?

Reply:
Narrowing down the build to x64 only. It's all Microsoft Visual Studio's mangling, but I triggered the change from the UI.